### DISCUSSION TITLE
Android Notch Calculation similar to iOS SafeArea (DisplayCutout API)

### DISCUSSION THREAD

**Discussion by FreakyAli** [dotnet/maui#22764] [Q&A]
### Description

With Android P, Android has a similar API to iOS that lets you get the DisplayCutout, 
MAUI should add an API that lets you handle this similar to how we have on iOS.

More information on DisplayCutout:
https://stackoverflow.com/questions/51743622/how-to-handle-notchdisplay-cutout-in-android-api-lower-than-28
https://developer.android.com/develop/ui/views/layout/display-cutout

### Public API Changes

XAML 

```xml      
xmlns:droid="clr-namespace:Microsoft.Maui.Controls.PlatformConfiguration.AndroidSpecific;assembly=Microsoft.Maui.Controls"
droid:Page.UseDisplayCutout="False"   
```

C#

```csharp
using Microsoft.Maui.Controls.PlatformConfiguration;
using Microsoft.Maui.Controls.PlatformConfiguration.AndroidSpecific;

On<Android>().SetUseDisplayCutout(false);
```

### Intended Use-Case

The best way to show what this would do i guess 

![image](https://github.com/dotnet/maui/assets/31090457/66b8efae-6937-47f1-b26d-872f1e8ca9f1)

**Comment by FreakyAli** [comment]
@PureWeen any context why this was turned into a discussion

**Reply by gitown** [reply]
I support this request, it would avoid us the hassle of implementing workarounds!

**Comment by FreakyAli** [comment]
@PureWeen  @samhouts  @jfversluis this was a valid feature request in my opinion but it was converted to a discussion can we know why?

**Reply by PureWeen** [reply]
It's easier to discuss new feature proposals using the discussion features, and then once a spec has been decided on and agreed upon we can create an issue to schedule the work. We are hoping to create a more official safearea concept that's xplat for NET10, especially with the new edge to edge features that we're seeing people want to opt into. 

I don't think we'd want to implement DisplayCutout with a bool, we'd probably want to give users a bit more control over the different features

**Reply by jfversluis** [reply]
Additionally its basically a duplicate of #2342
